lib/Controller/RoomController.php

```diff
@@ -512,11 +512,10 @@ protected function formatRoomV2(Room $room, ?Participant $currentParticipant): a
 
 		// FIXME This should not be done, but currently all the clients use it to get the avatar of the user …
 		if ($room->getType() === Room::ONE_TO_ONE_CALL) {
-			$participants = $room->getParticipants();
+			$participants = json_decode($room->getName(), true);
```

Comment:
This fixes leaking if the other participant left the one-to-one conversation (in that case there will be no icon) :+1:

If this pull request is not backported, should it be fixed too (in a different way) in previous versions? It could be seen at least in the search results, which would not apply to previous versions, but I do not know if it happened too with the mobile apps.

Reply:
> This fixes leaking if the other participant left the one-to-one conversation (in that case there will be no icon) +1

This worked well before already?

> If this pull request is not backported

We will not backport this, search results are 20 only too. any known issue place?

Reply:
Ignore all that, I am not in my brightest day... :facepalm:

Yes, that worked as expected. Even if `getParticipants` would not return the other participant of a one to one call if that participant left the conversation the `name` property would have been set to that user ID when he left the conversation, so everything was fine. Moreover the `RoomController` should not affect the search results, so the search issue had nothing to do with that.

In any case, before the `thumbnailUrl` of the search result was empty if the other participant left the call and now it has the proper icon :+1: